pkg/istio/client.go

```diff
@@ -63,30 +67,59 @@ func (c *Client) getVirtualService(ctx context.Context) (*istioclientapi.Virtual
 	return virtualService, nil
 }
 
-func (c *Client) createVirtualService(ctx context.Context, watcher *v1alpha1.Watcher,
+func (c *Client) CreateVirtualService(ctx context.Context, watcher *v1alpha1.Watcher,
```

Comment:
```suggestion
func (c *Client) createVirtualService(ctx context.Context, watcher *v1alpha1.Watcher,
```

Reply:
Thanks for catching that, leftover from testing...